jobs/payment-jobs/tests/jobs/factory.py

```diff
@@ -89,6 +89,7 @@ def factory_invoice(payment_account: PaymentAccount, status_code: str = InvoiceS
         invoice.cfs_account_id = cfs_account_id
 
     invoice.save()
+    print(invoice.payment_method_code, invoice.invoice_status_code)
```

Comment:
print remove

Reply:
Will fix this in next PR as I am already changing this file for reconciliation